apps.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: home
-title: Apps supporting MPTCP
+title: Support in apps / OS
```

Comment:
```suggestion
title: Support in Apps and Operating Systems
```

Reply:
With this, the title goes on two lines:

![image](https://github.com/user-attachments/assets/f2b0bc3e-c235-4149-8582-8d039bd8ccb2)

I tried to keep it short for that reason, but that's probably fine to have it on two lines. WDYT?